src/elements/common/pagination/__tests__/__snapshots__/MarkerBasedPagination.test.js.snap

```diff
@@ -0,0 +1,617 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
These snapshot tests are pretty hefty. It's fine to keep them here for now, but eventually, I think it'd be good to replace them with visual regression tests.